CREDITS

```diff
@@ -0,0 +1,2 @@
+ownCloud Android (GNU GPL v2.0 https://github.com/owncloud/android/blob/master/LICENSE.txt)
+    Part of Sharing-related code in ShareActivity
```

Comment:
I think it makes more sense to have the credits right where we use the code. Besides, we already have lots of files in the repo root, and I would prefer to keep that clean.